mobile_back_apple/cpp/backend_coreml/BUILD

```diff
@@ -67,11 +68,12 @@ pbtxt2header(
     ],
 )
 
-cc_library(
+objc_library(
```

Comment:
I got the error
```
F ./flutter/cpp/backends/external.h:116] Unable to load symbol mlperf_backend_matches_hardware: dlsym(0xa2bfe5b0, mlperf_backend_matches_hardware): symbol not found
```
You may want to keep using `cc_library` instead of `objc_library`.

Reply:
We can't use Swift directly in C++, that's we we need to use Objective-C generated header for this. To be able to import that header, the file should have the mm extension (mix of ObjectiveC and c++). And in bazel we need to use objc-library rule to be able to add resources with mm extension. But I will check more on this.

Reply:
In my testing it works fine when replacing `objc_library` with `cc_library` and renaming `main.mm` to `main.cc`.
You can test the main binary with this command:
`cd mobile_back_apple/dev-utils && make coreml`

Reply:
Hey Anh! You are right. It started to work. Not sure, why I had that issue that made me use the objc_library flag. Changed to the cc_library, and renamed main.mm back to main.cc.